plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -75,42 +75,32 @@ test.describe( 'Add New Variable Product Page', () => {
 		await resetVariableProductTour( baseURL, browser );
 	} );
 
-	test( 'shows the variable product tour', async ( { page } ) => {
```

Comment:
This isn't really a test because we aren't asserting anything.  So for now, I'm going to eliminate it and move the check for the tour down into the two tests below.